src/ck-ldb/MetaBalancer.C

```diff
@@ -162,6 +162,12 @@ void MetaBalancer::init(void) {
   // to false so that it doesn't clear the handles.
   lb_in_progress = false;
 
+  // After the first reduction completes, we have a way to manually check for
```

Comment:
Is this supposed to be "we have a way to" or "we have to?"

Reply:
We have a way to. As in, we have a way to do it otherwise, so we can ignore the periodic method.